src/Features/Blockcore.Features.ColdStaking/UI/Pages/ColdStakeReceive.razor

```diff
@@ -0,0 +1,218 @@
+
+                                        <tr>
+                                            @* @onclick="ViewTransaction" > *@
+                                            <td><CopyToClipboard Text="@_address.Address" /> </td>
+                                            <td class="text-center">@_address.IsChange</td>
+                                            <td class="text-center">@_address.AmountConfirmed</td>
+                                            <td class="text-center">@_address.AmountUnconfirmed</td>
+                                        </tr>
+
+                                    }
+                                </tbody>
+
+                            }
+
+                            @if (_ShowTab_UnUsedAddress)
+                            {
+                                <tbody>
+                                    @foreach (var _address in @accountmodel.Addresses.Where(li => li.IsUsed == false && li.IsChange == false))
+                                    {
+                                        <tr>
+                                            @* @onclick="ViewTransaction" > *@
+                                            <td><CopyToClipboard Text="@_address.Address" /> </td>
+                                            <td class="text-center">@_address.IsChange</td>
+                                            <td class="text-center">@_address.AmountConfirmed</td>
+                                            <td class="text-center">@_address.AmountUnconfirmed</td>
+                                        </tr>
+
+                                    }
+                                </tbody>
+
+                            }
+
+                            @if (_ShowTab_IsChanged)
+                            {
+                                <tbody>
+                                    @foreach (var _address in @accountmodel.Addresses.Where(li => li.IsUsed = false && li.IsChange == true))
+                                    {
+                                        <tr>
+                                            @* @onclick="ViewTransaction" > *@
+                                            <td><CopyToClipboard Text="@_address.Address" /> </td>
+                                            <td class="text-center">@_address.IsChange</td>
+                                            <td class="text-center">@_address.AmountConfirmed</td>
+                                            <td class="text-center">@_address.AmountUnconfirmed</td>
+                                        </tr>
+                                    }
+                                </tbody>
+                            }
+                        </table>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+}
+
+@code {
+    [Parameter]
+    public string walletname { get; set; }
+    [Parameter]
+    public string accountname { get; set; }
+
+
+    public Boolean _ShowTab_UsedAddress = true;
```

Comment:
Is it due to Blazor that this syntax is used? Prefixed with "_"? Shouldn't it be "ShowTabUsedAddress"?

Reply:
yeah but the UI is a bit of a mess when it comes to keeping our convention I will fix this